bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/io/ChannelPool.java

```diff
@@ -180,7 +149,7 @@ public boolean isShutdown() {
   @Override
   public boolean isTerminated() {
     boolean allTerminated = true;
-    for (ManagedChannel managedChannel: channels.get()) {
+    for (ManagedChannel managedChannel: channels) {
```

Comment:
nit: 'managedChannel: ' -> 'managedChannel : '.  There should be a space after managedChannel.